.gitignore

```diff
@@ -67,3 +67,4 @@ yarn.lock
 **/*.d.cts
 !**/types/**/*.d.ts
 
+page-functions-test-case*out*.js
```

Comment:
Some unit tests store random output files in the `.tmp` directory. WDYT about moving this output there?